lerna-debug.log

```diff
@@ -0,0 +1,53 @@
+0 silly argv {
```

Comment:
Should this file be checked in?

Reply:
Nope, that was a mistake. I removed the file.